packages-exp/auth-exp/src/core/persistence/indexed_db.ts

```diff
@@ -16,20 +16,20 @@
  */
 
 import {
+  PersistedBlob,
   Persistence,
   PersistenceType,
-  PersistenceValue,
-  Instantiator
-} from '.';
+  PersistenceValue
+} from './';
 
-const STORAGE_AVAILABLE_KEY_ = '__sak';
+const _STORAGE_AVAILABLE_KEY = '__sak';
```

Comment:
should this live in index? it's also declare in browser

Reply:
Done